aws.yaml

```diff
@@ -785,9 +785,61 @@ kind: Release
 metadata:
   annotations:
     giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
-  name: v9.0.5
+  name: v9.0.6
 spec:
   state: active
+  date: 2020-06-02T11:00:00Z
+  apps:
+    - name: cert-exporter
+      version: 1.2.2
+    - name: chart-operator
+      version: 0.13.0
+    - name: cluster-autoscaler
+      componentVersion: 1.16.2
+      version: 1.1.4
+    - name: coredns
+      componentVersion: 1.6.5
+      version: 1.1.8
+    - name: kube-state-metrics
+      componentVersion: 1.9.5
+      version: 1.0.5
+    - name: metrics-server
+      componentVersion: 0.3.3
+      version: 1.0.0
+    - name: net-exporter
+      version: 1.7.1
+    - name: nginx-ingress-controller
+      componentVersion: 0.30.0
+      version: 1.6.9
```

Comment:
@tfussell @MarcelMue I'm creating a PR to this PR to update nginx to 1.6.12. Will test it and request your review.
If you want to ship 9.0.6 sooner, can't wait for that upgrade to be included, just do it, I'll create a follow up release.

Reply:
Integrated. Thanks @tfussell for help with testing!